gc/base/MemorySubSpaceSemiSpace.cpp

```diff
@@ -842,7 +842,7 @@ MM_MemorySubSpaceSemiSpace::checkSubSpaceMemoryPostCollectResize(MM_EnvironmentB
 			omrtty_printf("New space resize check:\n");
 		}
 
-		if (extensions->scavengerStats._gcCount == 1) {
+		if (extensions->incrementScavengerStats._gcCount == 1) {
```

Comment:
this code deals with cycleTimes.cycleEnd, which means it's meant to be invoked at cycle end
indeed, the main user of this from:

```
	if (lastIncrement) {
...
			_activeSubSpace->mainTeardownForSuccessfulGC(env);
```

by which time incrementScavengerStats._gcCount will be 2
(it's true even for the current code: scavengerStats._gcCount will be 2)

So this guard is probably wrong (not properly updated when Concurrent Scavenger was introduced a few years ago). 
But let's leave it as-is. We (I) can try to fix it later - would require some local testing.